### PR TITLE
feat: Adds Romanian

### PR DESCRIPTION
**Pull request by sergiubologa**

### CODE REVIEW

File: site/ro.njk [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+
+            <h2 class="text-3xl">Manifestul Auto-Carantinei</h2>
+
+            <p>
+                În lipsa unor tratamente bine studiate și a unui vaccin viabil pentru care va trebui să așteptăm cel 
+                puțin încă un an, singura metodă eficientă de a menține epidemia de coronavirus sub control e de a-i 
+                oferi virusului cât mai puține șanse de a se răspândi. Următoarea listă de acțiuni ce pot fi 
+                implementate, ordonată de la cele mai ușoare la cele mai eficiente în lupta împotriva pandemiei, ar 
+                trebui să servească ca un set de sugestii pentru oamenii care doresc să se alăture mișcării atunci 
+                când cei responsabili continuă să le neglijeze.
```
Comment (NiBu-dev) [inline site/ro.njk]: Please change "responsabili" to "iresponsabili" because otherwise it doe not make sense. :) 
[edit]

File: site/ro.njk [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+            </p>
+
+            <ol class="text-xl mb-12 list-decimal pl-12">
+                <li class="pb-2">Nu te panica, dar fii în alertă.</li>
+                <li class="pb-2">Spală-te pe mâini des și exersează tusea și strănutul corect.</li>
```
Comment (NiBu-dev) [inline site/ro.njk]: A better word would be "practică" instead of "exersează"

File: site/ro.njk [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+                <li class="pb-2">Încearcă să-ți atingi cât mai puțin posibil fața, incluzând aici gura, nasul și ochii.</li>
+                <li class="pb-2">Exersează distanțarea socială, fără îmbrățișări, fără sărutări sau pupici, fără strângeri de mână, fără a bate palma. Dacă e necesar, folosește alternative mai sigure.</li>
```
Comment (NiBu-dev) [inline site/ro.njk]: Same thing for "exerseaza":
A better word would be "practică" instead of "exersează"

**Review by NiBu-dev — CHANGES_REQUESTED** [review]
Good job! Please see the comments in my review. :1st_place_medal:

File: site/ro.njk (hunk 1 above)
Comment (sergiubologa) [reply site/ro.njk]: 🤦‍♂  Done.

File: site/ro.njk (hunk 2 above)
Comment (sergiubologa) [reply site/ro.njk]: Replaced. 👍

File: site/ro.njk (hunk 3 above)
Comment (sergiubologa) [reply site/ro.njk]: Done.

**Review by NiBu-dev — APPROVED** [review]

File: site/ro.njk [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+
+            <ol class="text-xl mb-12 list-decimal pl-12">
+                <li class="pb-2">Nu te panica, dar fii în alertă.</li>
+                <li class="pb-2">Spală-te pe mâini des și practică tusea și strănutul corect.</li>
+                <li class="pb-2">Încearcă să-ți atingi cât mai puțin posibil fața, incluzând aici gura, nasul și ochii.</li>
+                <li class="pb-2">Practică distanțarea socială, fără îmbrățișări, fără sărutări sau pupici, fără strângeri de mână, fără a bate palma. Dacă e necesar, folosește alternative mai sigure.</li>
+                <li class="pb-2">Nu participa la concerte, piese de teatru, evenimente sportive sau orice alte evenimente destinate maselor.</li>
+                <li class="pb-2">Abține-te să mergi la muzee, expoziții, cinematografe, cluburi sau alte locuri de distracții.</li>
+                <li class="pb-2">Stai departe de întâlniri sociale și evenimente, întâlniri de club, slujbe religioase sau petreceri private.</li>
+                <li class="pb-2">Redu călatoriile la minimum. Nu călătorii distanțe mari decât dacă e absolut necesar.</li>
```
Comment (ghost) [inline site/ro.njk]: „Nu călători„, cu un singur **„i”**

**Comment by flore2003** [comment]
I this ready to merge?

**Comment by ahunts** [comment]
@sergiubologa @purdel @nbumbu Would you like to be attributed on our contributor's page? If so, what name would you like us to use?

**Comment by ahunts** [comment]
also, I have two other Romanian translations. If you three wouldn't mind going through them to see if any of the phrases are better, that would be great! Thank you
[Română - Stay The Fuck Home!.docx](https://github.com/flore2003/staythefuckhome/files/4340669/Romana.-.Stay.The.Fuck.Home.docx)
[#StayTheFuckHome Website - Romanian Translation.docx](https://github.com/flore2003/staythefuckhome/files/4340670/StayTheFuckHome.Website.-.Romanian.Translation.docx)

**Comment by NiBu-dev** [comment]
@flore2003 it shall be ready to be merged after the grammar issue pointed by @purdel is fixed.

File: site/ro.njk (hunk 4 above)
Comment (sergiubologa) [reply site/ro.njk]: Done.

**Comment by sergiubologa** [comment]
@flore2003 done with code review. It's ready to be merged.

**Comment by sergiubologa** [comment]
@ahunts @flore2003 is there something else needed to merge this one?

**Comment by sergiubologa** [comment]
@phelmig can you please merge this one too?

**Comment by sergiubologa** [comment]
😢

**Pull request closed without merging**